src/main/java/com/powsybl/openloadflow/NetworkCache.java

```diff
@@ -316,47 +317,44 @@ public void onExtensionUpdate(Extension<?> extension, String attribute, Object o
                 return;
             }
 
-            boolean[] done = new boolean[1];
+            CacheUpdateResult result = CacheUpdateResult.unsupportedUpdate();
             if ("secondaryVoltageControl".equals(extension.getName())) {
                 SecondaryVoltageControl svc = (SecondaryVoltageControl) extension;
-                onSecondaryVoltageControlExtensionUpdate(svc, attribute, newValue, done);
+                result = onSecondaryVoltageControlExtensionUpdate(svc, attribute, newValue);
             }
 
-            if (!done[0]) {
-                reset();
-            }
+            processUpdateResult((Identifiable<?>) extension.getExtendable(), attribute, result);
         }
 
-        private void onSecondaryVoltageControlExtensionUpdate(SecondaryVoltageControl svc, String attribute, Object newValue, boolean[] done) {
+        private CacheUpdateResult onSecondaryVoltageControlExtensionUpdate(SecondaryVoltageControl svc, String attribute, Object newValue) {
```

Comment:
You don't have any element not found for extensions.

Reply:
Fixed